### DISCUSSION TITLE
Chocolatey Installations stopped working?

### DISCUSSION THREAD

**Discussion by BananamanCoolDude** [amidaware/tacticalrmm#1519] [General]
**Server Info (please complete the following information):**
 - OS: [e.g. Ubuntu 20.04, Debian 10] : Ubuntu 20.04.6
 - Browser: [e.g. Chrome, safari]: Chrome, firefox
 - RMM Version (as shown in the top left of web UI): v0.15.11

**Installation Method:**
  - [ ] Standard
  - [x] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI):
- Agent OS: [e.g. Win 10 v2004, Server 2012 R2]

**Describe the bug**
Recently noticed that Chocolatey installations don't work anymore, maybe someone has a fix? Tried to deploy multiple targets Winrar and Notepad++, and Python  Message appears but nothing is being installed. Ideas?

**To Reproduce**
1. Go to Software
2. Click on + Install software
3. Search for Winrar
4. yes to install

**Expected behavior**
Installation to happen. But nothing happens at all...

**Screenshots**
![image](https://github.com/amidaware/tacticalrmm/assets/102809616/db5b3aa3-06f7-43cd-951a-309c9569fb04)

**Comment by silversword411** [comment]
Works here with latest. Your agent has to be online at the time of install. You can see the status of the task in right click pending agent actions

![2023-05-11_201453 - brave_Tactical_RMM_-_Brave](https://github.com/amidaware/tacticalrmm/assets/13395348/d29b2bbb-f9eb-4e3e-a1ae-ae34eed21fba)

![2023-05-11_201440 - brave_Tactical_RMM_-_Brave](https://github.com/amidaware/tacticalrmm/assets/13395348/b45c0746-cb90-4092-88a9-15757b22a966)

Are you seeing something different, please provide screenshots

**Comment by BananamanCoolDude** [comment]
![image](https://github.com/amidaware/tacticalrmm/assets/102809616/d37a6cfc-c169-4baf-9384-239730c6c541)

All Actions are update related automations. :(

**Comment by silversword411** [comment]
Ok....you'll need to provide better context than that.

**Comment by BananamanCoolDude** [comment]
All of the agents are online at the point of running installation. Pending actions does not display anything related to installations only Automated updates happening.

**Comment by BananamanCoolDude** [comment]
![image](https://github.com/amidaware/tacticalrmm/assets/102809616/359f2ad2-418b-463b-905f-890d13bfe917)

**Comment by silversword411** [comment]
You're a stock install?

Follow the pictures.
Right-click agent
Pending agent actions
Choose show completed if it wasn't already showing

**Comment by BananamanCoolDude** [comment]
That's interesting.. 

![image](https://github.com/amidaware/tacticalrmm/assets/102809616/f0f5ac46-65a3-4ee8-aa4b-1903e477a958)

**Comment by BananamanCoolDude** [comment]
No idea why it cannot find Chocolateley on some devices, any idea how to deploy installation via the cmd line?

**Comment by silversword411** [comment]
Installing using chocolatey install docs with something like 
```bat
@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
pause
```

Might do it. But you might actually be having problems with buggy windows environment variables

https://docs.tacticalrmm.com/troubleshooting/#executable-file-not-found-in-path
